packages/theming/android-theme/.npmignore

```diff
@@ -0,0 +1,14 @@
+src
```

Comment:
As pointed out in #https://github.com/microsoft/fluentui-react-native/pull/643 You'll want to remove the first line here too. =)

Reply:
done :)